src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -49,10 +50,13 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.logging.Logger;
+import java.util.logging.Level;import java.util.logging.Logger;
```

Comment:
```suggestion
import java.util.logging.Level;
import java.util.logging.Logger;
```

Reply:
For some reason my IDE does this when it adds the imports. I don't know why. I also never understood the point of specifying every single class for most cases, as we are already depending on the whole library. But again, I am not a full time Java Dev, I will try to comply.